plugins/ssr/ssrequality.ml

```diff
@@ -141,16 +141,16 @@ let newssrcongrtac arg ist gl =
 
 (** 7. Rewriting tactics (rewrite, unlock) *)
 
-(** Coq rewrite compatibility flag *)
+  -(** Coq rewrite compatibility flag *)
 
-let ssr_strict_match = ref false
 
 let _ =
-  Goptions.declare_bool_option 
+  let ssr_strict_match = ref false
+  Goptions.declare_bool_option
     { Goptions.optname  = "strict redex matching";
       Goptions.optkey   = ["Match"; "Strict"];
       Goptions.optread  = (fun () -> !ssr_strict_match);
-      Goptions.optdepr  = false;
+      Goptions.optdepr  = true; (* noop *)
```

Comment:
Same here - deprecation should be separate from documentation as far as I'm concerned.